src/main/java/duke/task/Task.java

```diff
@@ -0,0 +1,51 @@
+package duke.task;
+
+import duke.Ui;
+
+public abstract class Task {
+    private boolean isMarked;
+    private String content;
+
+    public Task(String Task_content) {
+        this.content = Task_content;
+        this.isMarked = false;
+    }
+
+    public Task(Boolean isMarked, String Task_content) {
+        this.content = Task_content;
+        this.isMarked = isMarked;
+    }
+
+    public boolean isMarked() {
+        return isMarked;
+    }
+
+    public String get_content () {
+        return content;
+    }
```

Comment:
Perhaps you can used camel case naming convention here as per the coding standard!